### ISSUE TITLE
the errors dialog should support scrolling when there are many errors

### ISSUE THREAD

**Issue by devoncarew** [dart-lang/dart-pad#1121]
When there are many errors, it’s not possible to see them all in the errors dialog:

<img width="429" alt="Screen Shot 2019-06-20 at 11 36 32 AM" src="https://user-images.githubusercontent.com/1269969/59872958-cc40a980-934f-11e9-8a26-adc9424786ba.png">

We should probably do something like cap the dialog to a max height, and have the content scroll vertically if necessary.

**Comment by RedBrogdon** [comment]
@galeyang 

Something we might also consider is whether it's even helpful to show that many issues at once. We could instead show the top XX analyzer issues and then have a line below saying there are more coming once you make a fix to one of the visible ones.

**Comment by galeyang** [comment]
TLDR: I'm happy with the immediate proposed fix (scrollable messages as needed). We can track @RedBrogdon's note in another issue if you like.

----
From what I've seen in the UX study, I observed that nearly all 10 participants encountered the issue of not having a scrollbar when it's overflowing. One user specifically brought up this point.  Therefore, I think what @devoncarew suggests below is a good immediate fix.

> We should probably do something like cap the dialog to a max height, and have the content scroll vertically if necessary.

For @RedBrogdon 's note, it's an interesting idea! I'm not certain on how users would react to that. But I have a question: how can we only present the most accurate, top messages?

I observed a few times that only one little thing is wrong. However, users saw so many error messages from the analyzer. For example, P10 missed the closing bracket here:
![image](https://user-images.githubusercontent.com/5102864/60305733-6d040b80-98f2-11e9-969a-5ffdabc9a8ff.png)

But these were messages shown up --- most messages (to me) are the "side effects" of the root cause (missing closing bracket)
![image](https://user-images.githubusercontent.com/5102864/60305674-3ded9a00-98f2-11e9-8d6e-a5fd82e57ee6.png)

I'm a little concern about this scenario if we couldn't present the most accurate errors.

Also, in the study, I didn't received specific complains on the number of analyzer issues shown _at once_. Maybe a lower priority?
